pkg/serviced.default

```diff
@@ -79,6 +79,9 @@
 # Set the BACKUPS path for serviced backups
 # SERVICED_BACKUPS_PATH=/opt/serviced/var/backups
 
+# Set the LOG_PATH for serviced logs
```

Comment:
Should probably add some clarification. Something like  "for serviced access and audit logs. Note that regular serviced operational messages are written to journald."